controller/src/main/java/org/jboss/as/controller/ObjectListAttributeDefinition.java

```diff
@@ -174,6 +174,10 @@ protected void addAllowedValuesToDescription(ModelNode result, ParameterValidato
     }
 
     public final ObjectTypeAttributeDefinition getValueType() {
+        return getValueAttributeDefinition();
+    }
+
+    public final ObjectTypeAttributeDefinition getValueAttributeDefinition() {
```

Comment:
is it correct? Shouldn't this `@Override` the method in ListAttributeDefinition.java